### PR TITLE
Drop support for pre-a9lh entrypoints

### PR DESCRIPTION
**Pull request by TuxSH**
Those entrypoints are terribly outdated and shouldn't be used anymore outside sb9si.

As you can see this also removes the need for painful-to-maintain support cruft inside GM9.

### CODE REVIEW

File: source/fs/fsperm.c [hunk 1]
```diff
@@ -171,7 +168,7 @@ bool SetWritePermissions(u32 perm, bool add_perm) {
                 return false;
             break;
         case PERM_SYS_LVL3:
-            if (!ShowUnlockSequence(6, "!THIS IS YOUR ONLY WARNING!\n \nYou want to enable SysNAND\nlvl3 writing permissions.\n \nThis enables you to OVERWRITE\n%s", IS_SIGHAX ? "your B9S installation and/or\nBRICK your console!" : IS_A9LH ? "your A9LH installation and/or\nBRICK your console!" : "essential system files and/or\nBRICK your console!"))
+            if (!ShowUnlockSequence(6, "!THIS IS YOUR ONLY WARNING!\n \nYou want to enable SysNAND\nlvl3 writing permissions.\n \nThis enables you to OVERWRITE\n%s", IS_SIGHAX ? "your B9S installation and/or\nBRICK your console!" : "your A9LH installation and/or\nBRICK your console!"))
```
Comment (dark-samus) [inline source/fs/fsperm.c]: I think there's a typo here

Comment (AnalogMan151) [inline source/fs/fsperm.c]: Is the typo that this B9S only version is still referencing A9LH even though if they had it this version could not be run?

Comment (mariogamer2) [inline source/fs/fsperm.c]: It's still have a9lh binary @AnalogMan151

Comment (AnalogMan151) [inline source/fs/fsperm.c]: My apologies, then I cannot spot the typo.

Comment (mariogamer2) [inline source/fs/fsperm.c]: Me too...

**Comment by d0k3** [comment]
While this is appreciated, it would be much more of a hassle to bring back removed entrypoints than  to lug them around for a little more.

Thus, this instead of taking the pull request:
https://github.com/d0k3/GodMode9/commit/d97e754593b31ed1618272aca67dca883c43890f
This will be kept until the next release to see if there are any legit complaints.

Please leave this pull request open, I'll take it at a later point.

**Comment by linouxis9** [comment]
I wonder why they want payloads for old entrypoints be removed that badly..

**Comment by d0k3** [comment]
There are good reasons to get rid of these unstable entrypoints. Difficult to maintain and test... Outdated, a lof of functionality not being available. There may also be legit reasons to keep, so I'm doing this slow.

**Comment by fox8091** [comment]
Perhaps also remove usage instructions for old entrypoints from the readme as well?

**Comment by d0k3** [comment]
Entrypoints removed locally, thus closing this now. Thanks for this pull request!

**Pull request closed without merging**